binding/python/_orca.py

```diff
@@ -319,53 +519,128 @@ def synthesize_to_file(
         :param output_path: Absolute path to the output audio file. The output file is saved as `WAV (.wav)`
         and consists of a single mono channel.
         :param speech_rate: Rate of speech of the generated audio.
+        :param random_state: Random seed for the synthesis process.
+        :return: A sequence of OrcaWordAlignment objects representing the word alignments.
         """
 
-        c_synthesize_params = self._get_c_synthesize_params(speech_rate=speech_rate)
+        c_synthesize_params = self._get_c_synthesize_params(speech_rate=speech_rate, random_state=random_state)
+
+        c_num_alignments = c_int32()
+        c_alignments = POINTER(POINTER(COrcaWordAlignment))()
 
         status = self._synthesize_to_file_func(
             self._handle,
             text.encode("utf-8"),
             c_synthesize_params,
-            output_path.encode("utf-8"))
-        if status is not self.PicovoiceStatuses.SUCCESS:
-            raise self._PICOVOICE_STATUS_TO_EXCEPTION[status](
+            output_path.encode("utf-8"),
+            byref(c_num_alignments),
+            byref(c_alignments))
+        if status is not PicovoiceStatuses.SUCCESS:
+            raise _PICOVOICE_STATUS_TO_EXCEPTION[status](
                 message="Unable to synthesize speech",
                 message_stack=self._get_error_stack())
 
+        alignments = []
```

Comment:
noticed that the above `synthesize` method has the same block of code here for creating the `alignments` array, wondering if it makes sense to turn this into a function or something

Reply:
yes, much better!